sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/models/ChatChoice.java

```diff
@@ -104,13 +104,6 @@ public ContentFilterResultsForChoice getContentFilterResults() {
         return this.contentFilterResults;
     }
 
-    /*
```

Comment:
Is this expected? If yes, we should add a note in the CHANGELOG to note it is removed.

Reply:
I saw it is removed from spec, so we need to add a note to CHANGELOG 

https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/OpenAI.Inference/models/completions/chat_completions.tsp#L286

Reply:
- Removed `finish_details` field from `ChatChoice` class since service API version, `2024-07-01-preview`, does not return this field.